### DISCUSSION TITLE
Cancel XHR when song is stopped, or load is called with a new song.

### DISCUSSION THREAD

**Discussion by psavva** [katspaugh/wavesurfer.js#2905] [Q&A]
Wavesurfer.js version(s): 6.6.4
Browser and operating system version(s): Windows 10, Chrome.

Support Question:
Is it possible to cancel the xhr http 206 which downloads chunked mp3 songs?

I would like to have the list of songs and play button next to each.

If I click play on one song, it starts downloading and starts playing.

If I click on the second song, it continues to download the first song, hence wasting bandwidth.

How can I cancel the download of the first song?

I'm using the load() function to load the music.

**Comment by katspaugh** [comment]
In v6, you can call `wavesurfer.cancelAjax()`.

**Reply by psavva** [reply]
I will provide a StackBlitz, showing that cancelAjax has no effect, not does capturing the loading event.

Let me demo that as soon as I get home
